rt-shell/objects/obj_shell/Create_0.gml

```diff
@@ -317,18 +351,52 @@ function _confirm_current_suggestion() {
 	cursorPos = string_length(consoleString) + 1;
 }
 
+/*
+ * Returns true if the array contains any instances that match the provided element
+ * otherwise returns false
+ */
+function _array_contains(array, element) {
+	for (var i = 0; i < array_length(array); i++) {
+		if (array[i] == element) {
+			return true;
+		}
+	}
+	return false;
+}
+
+/// @param value
+/// @param min_input
+/// @param max_input
+/// @param min_output
+/// @param max_output
+function _remap(value, min_input, max_input, min_output, max_output) {
+	var _t = (value - min_input) / (max_input - min_input);
+	return lerp(min_output, max_output, _t);
+}
+
+/*
+// OLD SPLIT
```

Comment:
You can go ahead and remove the old split method entirely rather than leave it commented out here.